packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -78,12 +125,49 @@ export class LionProgressIndicator extends LocalizeMixin(LitElement) {
 
   connectedCallback() {
     super.connectedCallback();
-    this.setAttribute('role', 'status');
-    this.setAttribute('aria-live', 'polite');
+    this.setAttribute('role', 'progressbar');
+  }
+
+  /**
+   * Update aria labels on state change.
+   * @param {import('@lion/core').PropertyValues } changedProperties
+   */
+  updated(changedProperties) {
+    super.updated(changedProperties);
+
+    if (!this.indeterminate) {
```

Comment:
Yeah better, although I personally still prefer the early return:) 
- https://github.com/Shopify/web-configs/blob/main/packages/eslint-plugin/docs/rules/prefer-early-return.md
- https://medium.com/swlh/return-early-pattern-3d18a41bba8